composer.json

```diff
@@ -44,7 +44,7 @@
     },
     "extra": {
         "branch-alias": {
-            "dev-master": "2.0-dev"
+            "dev-master": "1.3-dev"
```

Comment:
i propose we start a 1.3 branch from the 1.2.0 tag. that way we don't revert what is in master already.